src/CodeFormatter.cs

```diff
@@ -87,12 +89,24 @@ internal static class CodeFormatter
 
                 var exitCode = 0;
 
-                if (saveFormattedFiles && !workspace.TryApplyChanges(formattedSolution))
+                if (saveFormattedFiles && !workspace.TryApplyChanges(formattedSolution.Solution))
                 {
                     logger.LogError(Resources.Failed_to_save_formatting_changes);
                     exitCode = 1;
                 }
 
+                if (exitCode == 0 && !string.IsNullOrWhiteSpace(reportPath))
```

Comment:
If `reportPath` is a path to a file (".\\logs\\formatting.json"), can we use that as the report path? otherwise, appending a default file name to path is desirable.